src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
+  height: $card-height;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__product-details {
+    width: $card-content-width-adjustment;
+    margin-inline: $padding-inline;
+    margin-bottom: $margin-bottom;
+
+    &-title {
+      height: 36px;
+      font-size: $font-size-medium;
+      font-weight: $font-weight-medium;
+      line-height: 1.5;
+      text-align: left;
+      color: $secondary-color;
+      margin-top: 0;
+      margin-bottom: 4px;
+    }
+
+    &-description {
+      margin: 0;
+      font-size: $font-size-small;
+      font-weight: $font-weight-regular;
+      line-height: 14px;
```

Comment:
The line-height for the description seems to be a fixed pixel value which might not scale well with different font sizes. It's generally recommended to use unitless values for line-height for better scalability with font size changes.